templates/cfp/form_workshop_include.html

```diff
@@ -5,7 +5,7 @@ <h3>What we're looking for</h3>
 
 <h3>What it's like</h3>
 <p>We'll give you one of our workshop tents for up to four hours to give a workshop on any subject. Each workshop seats 30 people and comes equipped with a screen and speakers &ndash; we can also provide some equipment (such as soldering irons) on request.</p>
-<p>Workshops operate on a first-come-first-served basis to prevent problems with people not showing up &ndash; they are extremely popular and usually oversubscribed!</p>
+<p>We plan to operate an online sign-up system for workshops this year, with some tickets held back until closer to the start of the session, and a small queue for spaces left by no-shows. The workshops are always extremely popular and usually oversubscribed, and we hope this will prevent long queues and allow people to plan their day.</p>
```

Comment:
This could be an accidental US-English by me, but I wonder whether saying `a small waitlist for spaces left by no-shows` could be clearer.

I suggest it because I was trying to think of a way to avoid using the word `queue` in both sentences.  Again a nitpick, feel free to disregard.

Reply:
Actually I'm less keen on it now after thinking about it for a few more moments: `waitlist` could give the mistaken impression that there's an actual recorded list (either online, or prior to the event) to attend the event.  Queue it is.